packages/core/src/vars.css

```diff
@@ -150,32 +129,27 @@
 	--ts-border-color-focus: var(--ts-color-blue);
 	--ts-border-color-primary: var(--ts-color-blue);
 	--ts-border-color-disabled: var(--ts-color-gray-lighter);
-
 	--ts-border: var(--ts-border-width) var(--ts-border-type) var(--ts-border-color-default);
 	--ts-border-error: var(--ts-border-width) var(--ts-border-type) var(--ts-border-color-error);
 	--ts-border-success: var(--ts-border-width) var(--ts-border-type) var(--ts-border-color-success);
 	--ts-border-focus: var(--ts-border-width) var(--ts-border-type) var(--ts-border-color-focus);
 	--ts-border-disabled: var(--ts-border-width) var(--ts-border-type) var(--ts-border-color-disabled);
-
 	--ts-boxshadow-border-width: inset 0 0 0 var(--ts-border-width);
 	--ts-boxshadow-border-width-focused: inset 0 0 0 var(--ts-border-width-focused);
 	--ts-boxshadow-border-default: var(--ts-boxshadow-border-width) var(--ts-border-color-default);
 	--ts-boxshadow-border-error: var(--ts-boxshadow-border-width) var(--ts-border-color-error);
 	--ts-boxshadow-border-success: var(--ts-boxshadow-border-width) var(--ts-border-color-success);
 	--ts-boxshadow-border-focus: var(--ts-boxshadow-border-width) var(--ts-border-color-focus);
 	--ts-boxshadow-border-primary: var(--ts-boxshadow-border-width) var(--ts-border-color-primary);
-
 	--ts-progress-bar-height: 8px;
-
 	--ts-box-width-small: 280px;
 	--ts-box-width-medium: 360px;
-
 	--ts-shadow-color: var(--ts-color-slate-dark);
-
 	/* Spinner........................................................ */
+	--spinner-border-color: rgba(203, 215, 220, 0.5);
```

Comment:
Can you add a comment to say what's the color name? /* gray-lightest*/ or something like that!

Reply:
👍